### PR TITLE
Lint project

### PR DESCRIPTION
**Pull request by FireIsGood**
Adds prettier and lints the project

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,58 +1,60 @@
...
-        "mdast-util-toc": "^7.0.0",
-        "remark": "^15.0.1",
-        "unist-util-visit": "^5.0.0"
-    },
-    "optionalDependencies": {
-        "@rollup/rollup-linux-x64-gnu": "4.6.1"
-    }
-}
\ No newline at end of file
+  "name": "osuapp-website",
```
Comment (Nyumat) [inline package.json]: Can we keep it at four? (tab width)

File: package.json [hunk 2]
```diff
@@ -1,58 +1,60 @@
...
+  "type": "module",
+  "version": "0.0.1",
+  "scripts": {
+    "dev": "astro dev",
+    "start": "astro dev",
+    "build": "astro build",
+    "preview": "astro preview",
+    "astro": "astro"
+  },
```
Comment (Nyumat) [inline package.json]: maybe add a format script?

**Review by Nyumat — COMMENTED** [review]
minor stuff but hell yeah lgtm. essentially, defaults are cool (except tab width)

**Comment by FireIsGood** [comment]
Whoops, the revert commit also adds in the `format` script.

**Review by Nyumat — APPROVED** [review]

**Pull request merged**